kraken-app/kraken-app-hub/src/main/resources/mock/product.yaml

```diff
@@ -55,7 +61,20 @@ spec:
     #- classpath:/mock/api-targets-mappers/api-target-mapper.product.offering.eline.add.yaml
     #- classpath:/mock/api-targets-mappers/api-target-mapper.product.offering.uni.read.yaml
     #- classpath:/mock/api-targets-mappers/api-target-mapper.product.offering.eline.read.yaml
-    - classpath:/mock/mapping-matrix/mapping.matrix.order.yaml
-    - classpath:/mock/mapping-matrix/mapping.matrix.quote.yaml
-    - classpath:/mock/mapping-matrix/mapping.matrix.address.validation.yaml
-
+    #- classpath:/mock/api-targets-mappers/api-target-mapper.inventory.eline.list.yaml
```

Comment:
Please consider to remove commented out code

Reply:
Inventory is in use now. So this line will be reserved for fast testing in future.